tools/gen_stubs.zig

```diff
@@ -167,6 +171,7 @@ const MultiSym = struct {
             .{ .powerpc, 1 },
             .{ .powerpc64, 2 },
             .{ .aarch64, 2 },
+            .{ .loongarch64, 2 },
```

Comment:
What do the values in this array represent / where do they come from?

Reply:
I didn't check the details but my understanding is as the following.

This script will check whether a symbol exists in all architectures in the array.
If it exists in all architecture, it will be a `.globl` symbol, or it will be a `.weak` symbol.

Reply:
Well, I can't say I fully get what this function is doing, but the value seems to be consistent with the others (1 for 32-bit, 2 for 64-bit), so I guess it's fine? 🤷

Reply:
I went and familiarized myself a bit more with the code. The values here are `STB_GLOBAL` (1) and `STB_WEAK` (2). So yes, this is correct.